test-ts/manager.test.ts

```diff
@@ -283,7 +392,86 @@ describe("Manager", () => {
     });
   });
 
-  describe("#deposit()", async () => {
+  describe("#deprecateUnhealthyGroup()", () => {
```

Comment:
Can we add one happy path test ?

Reply:
Seems like all test were happy. I did add an unhappy one.😀 One that reverts when trying to deprecate a healthy group.